components/mpas-ocean/bld/build-namelist

```diff
@@ -936,10 +936,12 @@ if ($ocn_bgc eq 'eco_only' || $ocn_bgc eq 'eco_and_dms' || $ocn_bgc eq 'eco_and_
 if ($ocn_co2_type eq 'constant' ) {
         add_default($nl, 'config_ecosys_atm_co2_option', 'val'=>"constant");
         add_default($nl, 'config_ecosys_atm_alt_co2_option', 'val'=>"constant");
+        add_default($nl, 'config_ecosys_atm_alt_co2_use_eco', 'val'=>".false.");
         add_default($nl, 'config_ecosys_atm_co2_constant_value', 'val'=>$atm_co2_const_val);
 } else {
         add_default($nl, 'config_ecosys_atm_co2_option', 'val'=>"diagnostic");
         add_default($nl, 'config_ecosys_atm_alt_co2_option', 'val'=>"diagnostic");
+        add_default($nl, 'config_ecosys_atm_alt_co2_use_eco', 'val'=>".true.");
```

Comment:
is it expected to change ecosystem flags in this PR?

Reply:
No, I didn't think so either. That change was in the mpas-ocean Registry_ecosys.xml file and was picked up by the scripts that auto-generate part of the corresponding E3SM bld scripts. We'll have to involve @mark-petersen in fixing this.

Reply:
Strange. I did a white space clean-up on https://github.com/MPAS-Dev/MPAS-Model/commit/acf23bbd4ea6de99c250702dccb4095b58900e79 (search for  `config_ecosys_atm_alt_co2_use_eco`)  where I changed spaces to tabs in front of that flag definition, which is what it is supposed to be. The scripts apparently only process the flags with the correct tab formatting, which is why it is shown as an addition now.

I think that the MPAS-Ocean file 
`src/core_ocean/tracer_groups/Registry_ecosys.xml`
is now correct, and the above changes should have been in the build-namelist file all along. Since it was not, it was always using the default value of true before for E3SM runs.

@maltrud do you agree with the above false/true settings?

We could either leave these two lines in this PR (and the one below in `build-namelist-section`), or make a separate E3SM PR. I'm fine with either.

Reply:
Whoops, I missed that this is the one for maint-1.0. It probably doesn't matter either way, then, since that branch is not for BGC sims.

Reply:
@vanroekel - can you please look at the changes and add your review? I'd like to get this committed before July 8 when the high-res run may restart....